### DISCUSSION TITLE
Please help: perMANOVA using adonis2 on RNA-Seq data

### DISCUSSION THREAD

**Discussion by echo4922** [vegandevs/vegan#594] [Q&A]
I am a beginner and I am trying to perform a perMANOVA test on RNA-Seq data.
Because it is not an ecology data, the snippet of RNA-Seq data looks like the following:
![image](https://github.com/vegandevs/vegan/assets/112420424/1376ccd1-af6e-42b8-8383-e9bd7ce98fd0)

where the first column is gene id then three biological replicates for control (column 2 to column 4) then three biological replicates for treatment (column 5 to 7). Please note that the RNA-seq data has been log transformed.

Looking at Vegan manual,  dune.env acts as meta data for data dune.
![image](https://github.com/vegandevs/vegan/assets/112420424/2ada9c71-cdba-4334-af15-582642de403f)

My question is how should I format my RNA-seq data such that I can create dune.env data as meta data to perform perMANOVA analysis? 
Thank you.

**Comment by gavinsimpson** [comment]
Your data needs to be transposed relative to how you have it. You want genes in the columns and samples in the rows. That would be the equivalent of `dune`

You should also create another sheet with the sample ids in column 1 and the treatment info in column 2 and rep in column 3 for example. This would be the equivalent of `dune.env`.

In both cases you want the sample ID information to be stored as the rownames of the two respective data frames.

Other than that, consult any good introduction to base R to see how to read the data into R (that's not a vegan issue).
